src/main/java/uk/gov/hmcts/reform/roleassignmentbatch/processors/EntityWrapperProcessor.java

```diff
@@ -20,31 +26,50 @@
      * @param ccdCaseUsers to be processed
      * @return potentially modified or new item for continued processing, {@code null} if processing of the
-     * provided item should not continue.
+     *     provided item should not continue.
      * @throws Exception thrown if exception occurs during processing.
      */
     @Override
     public EntityWrapper process(CcdCaseUsers ccdCaseUsers) throws Exception {
         UUID requestUuid = UUID.randomUUID();
-        Newtable newtable = Newtable.builder().myid(requestUuid.toString()).column2(requestUuid.toString()).build();
+
         RequestEntity requestEntity = RequestEntity.builder()
-                                                   .id(requestUuid)
-                                                   .correlationId(UUID.randomUUID().toString())
-                                                   .clientId("ccd_migration")
-                                                   .authenticatedUserId("A fixed Authenticated User Id")
-                                                   .assignerId(ccdCaseUsers.getUserId())
-                                                   .requestType("CREATE")
-                                                   .status("APPROVED")
-                                                   .process("CCD")
-                                                   .replaceExisting(false)
-                                                   .roleAssignmentId(UUID.randomUUID())
-                                                   .reference(ccdCaseUsers.getCaseDataId().concat(ccdCaseUsers.getUserId()))
-                                                   .log(null)
-                                                   .created(LocalDateTime.now())
-                                                   .build();
+                                               .id(requestUuid)
+                                               .correlationId(UUID.randomUUID().toString())
+                                               .clientId("ccd_migration")
+                                               .authenticatedUserId("A fixed Authenticated User Id")
+                                               .assignerId(ccdCaseUsers.getUserId())
+                                               .requestType("CREATE")
+                                               .status("APPROVED")
+                                               .process("CCD")
+                                               .replaceExisting(false)
+                                               .roleAssignmentId(UUID.randomUUID())
+                                               .reference(ccdCaseUsers.getCaseDataId().concat(ccdCaseUsers.getUserId()))
+                                               .log(null)
+                                               .created(LocalDateTime.now())
+                                               .build();
+
+        HistoryEntity historyEntity = HistoryEntity.builder()
+                .id(requestUuid)
```

Comment:
why are we keeping the history entity Id same as requestId?

Reply:
This is sample builder to verify all the data is persisting or not. Nitish is doing these builder on his branch.

Reply:
ok, I need to then check the main branch builder.